### ISSUE TITLE
Regression with mean absolute error metric: MyDummyRegressor

### ISSUE THREAD

**Issue by CoodieDyl** [automl/auto-sklearn#528]
I'm trying to use the Python library auto-sklearn, based on the library scikit-learn, for a regression problem.
When I follow the tutorials on the official documentation, everything works fine, but when I want to change the metric from r2 to mean absolute error, I only have "MyDummyRegressor" as output.
Here is my code
 ```
def main():
X, y = sklearn.datasets.make_regression()                                                          
X_train, X_test, y_train, y_test = \                                                               
    sklearn.model_selection.train_test_split(X, y, random_state=1)
automl = autosklearn.regression.AutoSklearnRegressor(
    time_left_for_this_task=120,
    per_run_time_limit=30,
    tmp_folder='/tmp/autosklearn_holdout_example_tmp',
    output_folder='/tmp/autosklearn_holdout_example_out',
    disable_evaluator_output=False,
    ml_memory_limit=1024,
    seed=123,
)
automl.fit(
    X_train,
    y_train,
    dataset_name='random_regression',
    metric=autosklearn.metrics.mean_absolute_error)

print(automl.show_models())
test_predictions = automl.predict(X_test, batch_size=100)
print("MAE test score:", sklearn.metrics.mean_absolute_error(y_test, test_predictions))

print(automl.sprint_statistics())
```

And here is the output

```
[(1.000000, MyDummyRegressor(configuration=1, init_params=None, random_state=None)),]
MAE test score: 160.38897901387986
auto-sklearn results:
  Dataset name: random_regression
  Metric: mean_absolute_error
  Best validation score: 0.000000
  Number of target algorithm runs: 88
  Number of successful target algorithm runs: 50
  Number of crashed target algorithm runs: 37
  Number of target algorithms that exceeded the time limit: 0
  Number of target algorithms that exceeded the memory limit: 1
```

So basically, I have an ensemble of results with the r2 performance metric, but not with the mean absolute error, and no other as well as I tried them.

**Comment by mfeurer** [comment]
Thanks for reporting this, I think this is bug #500. We're already working on a fix with #522.

**Comment by mfeurer** [comment]
This should be fixed in the latest release 0.4.1.
